sqrl-planner/src/main/java/com/datasqrl/loaders/ObjectLoaderImpl.java

```diff
@@ -45,15 +47,19 @@ public class ObjectLoaderImpl implements ObjectLoader {
   private final CalciteTableFactory tableFactory;
   private final ModuleLoader moduleLoader;
   private final TableConfigLoader tableConfigFactory;
+  private final PackageJson sqrlConfig;
+  private final LogManager logManager;
 
   public ObjectLoaderImpl(ResourceResolver resourceResolver, ErrorCollector errors,
       CalciteTableFactory tableFactory, ModuleLoader moduleLoader,
-      TableConfigLoader tableConfigFactory) {
+      TableConfigLoader tableConfigFactory, PackageJson sqrlConfig, LogManager logManager) {
     this.resourceResolver = resourceResolver;
     this.errors = errors;
     this.tableFactory = tableFactory;
     this.moduleLoader = moduleLoader;
     this.tableConfigFactory = tableConfigFactory;
+    this.sqrlConfig = sqrlConfig;
+    this.logManager = logManager;
```

Comment:
object loader is just for loading modules. it shouldn't be aware of a log manager?

Reply:
Yes, I agree in the previous PR it was better when we just injected the LogManager into the ScriptPlaner. However since that object is created manually I don't see any other option to inject these classes into it.

Reply:
@henneberger Do you have a better idea for the modularity boundary here? I'm not a big fan of the PackageJson and the LogManager being in the ScriptPlanner. In particular now that it cannot be injected anymore.
Should the LogManager live in the Schema (since it's essentially a schema element)?
Where should the configuration live so the ScriptPlanner has access to it?